index.html

```diff
@@ -148,15 +197,11 @@ <h2>Get started</h2>
       <article>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
         dolore magna.</article>
       <form action="https://formspree.io/f/mjvdryev" method="POST">
-        <div class="the-relative">
-          <input type="text" maxlength="30" placeholder="Your name" required>
-        </div>
-        <div class="the-relative">
-          <input type="email" placeholder="Your email" required>
-        </div>
-        <div class="the-relative">
-          <input type="text" maxlength="500" class="text-area" placeholder="Enter your message here.." required>
-        </div>
+        <ul class="the-relative">
+         <li class="tname"><input type="text" maxlength="30" placeholder="Your name" required></li> 
+         <li class="mail"><input type="email" placeholder="Your email" required></li> 
+         <li class="texts"><input type="text" maxlength="500" class="text-area" placeholder="Enter your message here.." required></li> 
+        </ul>
```

Comment:
- [ ] Good Job on your UI design. Kindly ensure that your form input field matches the figma design 👍🏽. Please take care of those thick visible borders 😄 

your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/222928053-4af53a5f-f6a9-4205-acae-1d23fb85534f.png)

FIgma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/222928081-a9cebb6b-f535-43d6-93a2-85165805149f.png)